pkgs/applications/editors/vim/plugins/overrides.nix

```
@@ -1812,6 +1812,13 @@ in
     nvimRequireCheck = "refactoring";
   };
 
+  remote-sshfs-nvim = super.remote-sshfs-nvim.overrideAttrs {
+    dependencies = with self; [
+      telescope-nvim
+      plenary-nvim
+    ];
```

You can add a build time check whether the plugin can be imported with lua `require`.

```suggestion
    ];
    
    nvimRequireCheck = "remote-sshfs";
```